src/core/util/jsonQuery.js

```diff
@@ -180,6 +203,14 @@ function __wrapIndex(idx, arr) {
     return idx;
 }
 
+/**
```

Comment:
This method (and all others of this files are looking strange in the generated jsdoc -> It starts with "(inner) [jsonRequery.first]" and parameters are not formatted in italic.

Reply:
There's some issue with the jsdoc generation of arrow functions: @function and @description tags do not work as expected.